### DISCUSSION TITLE
Select-Init discrepancies in case of caching of prices

### DISCUSSION THREAD

**Discussion by BLR-0118** [ONDC-Official/protocol-network-extension#22] [General]
In some cases, seller apps may cache the prices for the /select response (for better performance) but will update with real-time price in /init. If this happens, the buyer will be taken back to the cart and the /select will be called again and this will create an continuous loop between /select & /init. How should this be handled?
@aakasjayaraj @abaweja @batramanuj @mahoriR @mherle @EkanshKr @VidyaGF @nitinmish @ONDC-Official/participants

**Comment by mherle** [comment]
when the realtime price update happens (in init), if the caching entity can update the cache, it should resolve the issue, right ?
